src/index.js

```diff
@@ -217,8 +217,13 @@ module.exports = (
         if (err) return reject(err);
         compiler.close(err => {
           if (err) return reject(err);
-          if (stats.hasErrors())
-            return reject(new Error(stats.toString()));
+          if (stats.hasErrors()) {
+            let errLog = '';
+            stats.compilation.errors.forEach((err, index) => {
```

Comment:
This is an interesting way to do it. I would have done

```
stats.compilation.errors.map(err => err.message).join('\n')
```

But I guess yours only loops once?

Reply:
Sure, let's keep it simple.